knimin/lib/data_access.py

```diff
@@ -2415,6 +2415,126 @@ def set_deposited_ebi(self):
                  WHERE barcode IN %s"""
         self._con.execute(sql, [barcodes])
 
+    def create_study(self, study_id, title=None, alias=None, notes=None):
+        """Creates a study
+
+        Parameters
+        ----------
+        study_id : int
+            Positive if it is a Qiita study ID, negative if others
+        title : str
+        alias : str
+        notes : str
+
+        Raises
+        ------
+        ValueError
+            If there is error creating the study
+        """
+        with self._con.cursor() as cur:
+            cur.execute('BEGIN')
```

Comment:
I believe this will establish the transaction. @josenavas, I do think you're the resident expert :) any concerns with isolation level or implicit behavior from executions which get run inside (e.g., the calls to `execute_fetchone`)?